atlasdb-api/src/main/java/com/palantir/atlasdb/config/LeaderConfig.java

```diff
@@ -47,15 +49,13 @@ public File acceptorLogDir() {
     public abstract String localServer();
 
     @Size(min = 1)
-    public abstract Set<String> leaders();
+    public abstract List<String> leaders();
```

Comment:
See my comment below in `check()` -- we can still verify correct configuration based on unique potential leaders and desired quorum size, as well as provide more useful error messages when folks misconfigure things.